test/media/drm_engine_unit.js

```diff
@@ -163,6 +163,22 @@ describe('DrmEngine', () => {
     });
   });
 
+
+  describe('createOrLoad', () => {
+    it('does not hang when given empty init data for offline', async () => {
+      tweakDrmInfos((drmInfos) => {
+        // An empty uint8array, like we make for fairplay content.
+        drmInfos[0].initData = [
+          {initData: new Uint8Array(0), initDataType: 'cenc', keyId: null},
+        ];
+      });
+      await drmEngine.initForStorage(
+          manifest.variants, /* usePersistentLicenses= */ false);
+      await drmEngine.createOrLoad();
+      expect(drmEngine.initialized()).toBe(true);
+    }, /* timeout= */ 100);
```

Comment:
With such a small timeout, I think you need a test run in the lab before merging.  That seems like it could flake easily on some platforms.

Maybe you can also bump this to 1000ms.  I'm not sure it needs to be so aggressive.

Reply:
Made it less aggressive.